src/components/PostInfo/PostInfo.jsx

```diff
@@ -1 +1,24 @@
-export const PostInfo = () => <>Put the post here</>;
+import './PostInfo.scss';
+import { CommentList, NoComments } from '../CommentList';
+import { UserInfo } from '../UserInfo';
+
+function checkComments(postComments) {
+  if (postComments.length !== 0) {
+    return <CommentList comments={postComments} />;
+  }
+
+  return <NoComments />;
+}
+
+export const PostInfo = ({ post }) => (
+  <div className="PostInfo">
+    <div className="PostInfo__header">
+      <h3 className="PostInfo__title">{post.title}</h3>
+
+      <UserInfo user={post.user} />
```

Comment:
let's render the Userinfo component only when user exists
```suggestion
      {post.user && <UserInfo user={post.user} />}
```